docs/src/external_links.md

```diff
@@ -1,7 +1,17 @@
+!!! tip
+    If you have an article, project, tool, or anything related to TiTiler that is not yet listed here, create a [Pull Request adding it](https://github.com/developmentseed/titiler/edit/main/docs/src/external_links.md).
 
+## eoAPI
 
-!!! tip
-    If you have an article, project, tool, or anything related to TiTiler that is not yet listed here, create a [Pull Request adding it](https://github.com/developmentseed/titiler/edit/main/docs/external-links.md).
+[eoAPI](https://eoapi.dev/) is a modular and cloud-native set of services for serving STAC-compliant geospatial data. It builds on the same ecosystem as TiTiler and aims to make deploying and scaling Earth observation APIs easier.
```

Comment:
To me the link with titiler is quite vague. I think we should clearly say that titiler is part of the building block of the eoapi stack.